pandas/core/construction.py

```diff
@@ -549,3 +549,38 @@ def _try_cast(
         else:
             subarr = np.array(arr, dtype=object, copy=copy)
     return subarr
+
+
+# see gh-17261
+def is_empty_data(data):
+    """
+    Utility to check if a Series is instantiated with empty data
+    """
+    is_none = data is None
+    is_simple_empty = isinstance(data, (list, tuple, dict)) and not data
+    return is_none or is_simple_empty
+
+
+def create_series_with_explicit_dtype(
+    data=None,
+    index=None,
+    dtype=None,
+    name=None,
+    copy=False,
+    fastpath=False,
+    dtype_if_empty=object,
+):
+    """
+    Helper to pass an explicit dtype when instantiating an empty Series.
```

Comment:
can you add a full doc-string (e.g. the paramters section)

Reply:
There are just the same as Series, so I wouldn't copy the full docstring, but just document the additional keyword (`dtype_if_empty`) and refer to Series docstring for the others

Reply:
>  refer to Series docstring for the others

which is actually already done